api/swagger-spec/oapi-v1.json

```diff
@@ -18214,6 +18217,10 @@
       "$ref": "v1.RollingDeploymentStrategyParams",
       "description": "input to the Rolling deployment strategy"
      },
+     "after": {
+      "$ref": "v1.AfterDeploymentActions",
```

Comment:
Can't we just stick to Pre/Post names we already have at Strategy levels. The only difference is - those will be at Deployment level and not at strategy and explain to users how these relate.